Templates/colors.json

```diff
@@ -4656,31 +4656,31 @@
     "EnvSound": {
       "struct": "ObjEnvSoundSpawner",
       "categories": [
-        "サウンド"
```

Comment:
The katakana for `Sound` has been translated to its English equivalent to make it easier for people to read it in tools interacting with this file. The original does use katakana for this single category.